geoips/plugins/modules/output_checkers/image.py

```diff
@@ -51,9 +51,8 @@ def outputs_match(plugin, output_product, compare_product, threshold="lenient"):
         Current output product
     compare_product : str
         Path to comparison product
-    threshold : str, optional
-        "theshold" argument to allow small diffs to pass - larger "threshold" factor
-        to make comparison less strict, by default 5%.
+    threshold: float
+        Threshold for the image comparison. Argument to pixelmatch.
```

Comment:
This is where we probably want to fully document what the threshold arg means.  Maybe at least say it should be between 0 and 1.  ALSO this is where we should consider where the default should be set, etc, since it is in both the "def call" and "def outputs_match" call signatures. @jsolbrig

Reply:
I added a note to the CLI Issue: https://github.com/NRLMMD-GEOIPS/geoips/issues/37

And I updated the docstring to at least be a little useful, feel free to resolve if you are good with it.